examples/functions/direxists/function.tf

```diff
@@ -0,0 +1,16 @@
+# Configuration using provider functions must include required_providers configuration.
+terraform {
+  required_providers {
+    local = {
+      source = "hashicorp/local"
+      # Setting the provider version is a strongly recommended practice
+      # version = "..."
+    }
+  }
+  # Provider functions require Terraform 1.8 and later.
+  required_version = ">= 1.8.0"
+}
+
+output "example_output" {
+  value = provider :: local :: direxists("${path.module}/example-directory")
```

Comment:
Same formatting note as the time provider, will be fixed before merge by the beta release of Terraform. Good thing cause of the `local` syntax highlighting lol

Reply:
Fixed with introduction of beta release: 9c98a52dcaf6051daafd403f055b13ece5efa603